app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,36 @@
+class VotesController < ApplicationController
+  def index
+  end
+
+  def create
+    @review = Review.find(params[:review_id])
+    @site = @review.site
+    @vote = Vote.new(vote_params)
+    @vote.review = @review
+    @vote.user = current_user
+    # Instead of update method
+    begin
+      if @vote.save
+        redirect_to site_path(@site)
+        flash[:alert] = "Vote has been recorded"
+      else
+        redirect_to site_path(@site)
+        flash[:alert] = "Vote Already Recorded"
+      end
+    rescue
```

Comment:
Why is exception handling being utilized here? 

http://launchware.com/articles/catch-all-exception-handling-is-not-flow-control